src/riscv.h

```diff
@@ -7,6 +7,22 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#if RV32_HAS(EXT_F)
```

Comment:
Don't do that. The file `riscv.h` is supposed to be a public header, and there should be no implementation details.